package.xml

```diff
@@ -27,6 +27,12 @@
   <buildtool_depend>catkin</buildtool_depend>
 
   <build_depend>pkg-config</build_depend>
+  <build_depend>libx11-dev</build_depend>
```

Comment:
@tylerjw hello,
I'm not a ROS maintainer, but those X11-specific dependencies might create conflicts for ROS users who build and use the SDK with headless distros.

Reply:
This file is used in ROS when someone is doing a source build of this library.  In that case they need these dependencies to build.  Because I made them `build_depend` tags someone who is just installing the library as a debian package through ros won't need to install these as they are only needed for building to library.